tap/passive_tapper.go

```diff
@@ -429,6 +429,21 @@ func startPassiveTapper(harWriter *HarWriter, outboundLinkWriter *OutboundLinkWr
 			assemblerMutex.Lock()
 			assembler.AssembleWithContext(packet.NetworkLayer().NetworkFlow(), tcp, &c)
 			assemblerMutex.Unlock()
+
+			sourceIP := packet.NetworkLayer().NetworkFlow().Src().String()
+			destinationIP := packet.NetworkLayer().NetworkFlow().Dst().String()
+			destinationPort := int(tcp.DstPort)
+
+			if streamFactory.getStreamProps(sourceIP, destinationIP, destinationPort).isTapTarget {
+				fmt.Printf("%s -> %s:%d isTapTarget:%t", sourceIP, destinationIP, destinationPort, streamFactory.getStreamProps(sourceIP, destinationIP, destinationPort).isTapTarget)
+				if IsTLSHandshakePacket(tcp.Payload) {
```

Comment:
Won't there only be a handshake when the there's a new stream?
Why do we need to run this for every single tcp packet?
Also, is it possible for the handshake to be split between several tcp packets? In that case we would need to look for the handshake after tcp assembly.